web/src/interfaces.ts

```diff
@@ -57,11 +57,21 @@ export interface Listing {
   listingStatus: ListingStatus;
 }
 
-export interface Customer {
-  id: number;
+/**
+ * CustomerPayload Interface that contains the fields of the payload that
+ * would be sent to the server.
+ */
+export interface CustomerPayload {
   gpayId: string; // E164 format
```

Comment:
Why gpay ID has this format?

Reply:
@chshda Parallax error 😆 Should be for the contactNumber